.github/workflows/release-prepare-hotfix.yml

```diff
@@ -99,3 +99,44 @@ jobs:
       - name: push changes
         run: |
           git push origin ${{ inputs.ref }}
+
+      - name: fetch develop and main
+        if: ${{ github.event.inputs.tag_version == 'latest' }}
+        run: |
+          git fetch origin develop main
+
+      - name: merge into main
+        if: ${{ github.event.inputs.tag_version == 'latest' }}
+        run: |
+          git checkout main
+          git merge ${{ inputs.ref }} --no-ff
+          git push origin main
```

Comment:
so we merge to main without a PR like develop ?

Reply:
sorry the diffs here are misleading. I have literally copy-pasted the existing process that was removed [here](https://github.com/LedgerHQ/ledger-live/pull/8264/files#diff-f2d788d05c244de0769b76766bce63adf5c5ff170e6845ddd17c64ea6e89a91aL55), except added `if` statements

Reply:
ok it's out of your ticket scope i resolve the issue . Thanks